hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -1532,6 +1532,16 @@
     "hbase.regionserver.slowlog.buffer.enabled";
   public static final boolean DEFAULT_ONLINE_LOG_PROVIDER_ENABLED = false;
 
+  /** The slowlog info family as a string*/
+  private static final String SLOWLOG_INFO_FAMILY_STR = "info";
+
+  /** The slowlog info family */
+  public static final byte [] SLOWLOG_INFO_FAMILY = Bytes.toBytes(SLOWLOG_INFO_FAMILY_STR);
+
+  public static final String SLOW_LOG_SYS_TABLE_ENABLED_KEY =
+    "hbase.regionserver.slowlog.systable.enabled";
+  public static final boolean DEFAULT_SLOW_LOG_SYS_TABLE_ENABLED_KEY = false;
```

Comment:
On high level, it will be redundant but if we are looking for completeness of logs, this might provide better details. Because as of today, we log truncated data at RpcServer level. At times, we miss on huge region name.
```
e.g
"param":"region { type: REGION_NAME value: \"t1,\\000\\000\\215\\f)o\\\\\\024\\302\\220\\000\\000\\000\\000\\000\\001\\000\\000\\000\\000\\000\\006\\000\\000\\000\\000\\000\\005
000000<TRUNCATED>"
```

But regardless, this is not supposed to be enabled by default for sure. Ringbuffer feature is also so far disabled (by default). Maybe we can plan to enable it by default starting release 3.0.